src/recipes.ts

```diff
@@ -1,68 +1,99 @@
-import { v4 as uuidv4 } from 'uuid'
-import { Storage } from './utils'
+import axios from 'axios'
+import { ingredientList } from './ingredients'
+import { Recipe } from './types/recipe'
+import { Ingredient } from './types/ingredient'
+import { getDataFromDb, removeDataElement } from './utils'
 
-let recipes: Array<Object> = []
+const allRecipesPath = 'http://localhost:3001/recipes/all'
```

Comment:
you should use axios global config, and pass global url to to repeat yourself mutliple times